front-end/src/components/charts/chart-screen.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { StyledChartScreen } from './chart-screen.styles'
+import LineChart from './line-chart'
+
+const ChartScreen = () => {
+	return (
```

Comment:
this is not really javascript but probably should be either 2 or 4 space indentation. we haven't had jsx before, so let's decide on what it should be.